tensorflow/compiler/mlir/tosa/tfl_passes.cc

```diff
@@ -52,6 +53,11 @@ void createTFLtoTOSALegalizationPipeline(
   pm.addPass(mlir::affine::createLoopFusionPass());
   pm.addPass(mlir::affine::createAffineScalarReplacementPass());
 
+  if (opts.convert_attrs_to_emitc) {
```

Comment:
I don't think it makes sense as part of this pipeline as its not related to TOSA legalization. Why not just add this post this pipeline creation where needed?